readme.md

```diff
@@ -120,19 +120,26 @@ The first argument is an array of default actions that can be used. The second a
 
 `MenuItem` labels may contain the placeholder `{selection}` which will be replaced by the currently selected text as described in [`options.labels`](#labels).
 
+#### showLearnSpelling
+
+Type: `boolean`\
+Default: `true`
+
+Show the `Learn Spelling {selection}` menu item when right-clicking text.
```

Comment:
This makes it sound like it always shows, which is not true.

Reply:
You do have the info

> To get spellchecking, “Correct Automatically”, and “Learn Spelling” in the menu, please enable the `spellcheck` preference in browser window: `new BrowserWindow({webPreferences: {spellcheck: true}})

I thought it would be redundant, could rephrase it to this ?

> Show the `Learn Spelling {selection}` menu item when right-clicking text. The `spellcheck` preference in browser window must be enabled.

Reply:
I meant; it will only show when right-clicking misspelled words.

Reply:
Adjust with this to be precise ?

> Show the `Learn Spelling {selection}` menu item when right-clicking text. Even if `true`, the `spellcheck` preference in browser window must still be enabled. It will also only show when right-clicking misspelled words.